packages/@aws-cdk/aws-glue-alpha/README.md

```diff
@@ -219,7 +219,23 @@ new glue.Table(this, 'MyTable', {
 });
 ```
 
-By default, an S3 bucket will be created to store the table's data and stored in the bucket root. You can also manually pass the `bucket` and `s3Prefix`:
+However, if your data will not reside in a S3 bucket, you can use the `externalDataLocation` property to point to the data location:
+
+```ts
+declare const myDatabase: glue.Database;
+new glue.Table(this, 'MyTable', {
+  externalDataLocation: 'default_db_public_example', // A table in Redshift
+  // ...
+  database: myDatabase,
+  columns: [{
+    name: 'col1',
+    type: glue.Schema.STRING,
+  }],
+  dataFormat: glue.DataFormat.JSON,
+});
+```
+
+Using `externalDataLocation` will override the `bucket` and `s3Prefix` properties.
```

Comment:
As I understand it `externalDataLocation` only works together with `connection`, right?
If that's the case, your two new examples probably should be combined into one.

Reply:
You're right actually. I however read in the documentation that [nested properties should rather be flattened](https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md#flat)

Reply:
Okay, I'll get to work on this.

Reply:
This has been done